rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py

```diff
@@ -46,12 +47,17 @@ def __init__(self):
         self._node_initialized = False
         self._node = None
 
+    def __del__(self):
+        self._destroy_node()
```

Comment:
The node needs to be destroyed earlier since this destructor is not guaranteed to be called.

Reply:
Adding it to the derived method shutdown() instead